app/errors.py

```
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, name: str = "NotVaccinatedError") -> None:
+        super().__init__(name)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, name: str = "OutdatedVaccineError") -> None:
+        super().__init__(name)
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, name: str = "NotWearingMaskError") -> None:
```

This violates checklist item #3: 'Use descriptive error messages'. Please provide a descriptive default message, such as "Visitor must wear a mask."